Sources/SuperDelegate+LocalNotifications.swift

```diff
@@ -43,6 +43,7 @@ public protocol LocalNotificationActionCapable: LocalNotificationCapable {
 // MARK: - SuperDelegate Local Notification Extension
 
 
+@available(iOS, deprecated: 10.0)
```

Comment:
Doesn't this deprecate the entire class instead of just the methods in the extension?